src/javascripts/templates/transactionTemplate.js

```diff
@@ -19,36 +18,36 @@ class Template {
       const amountClass = isIncome ? 'income' : 'outflow';
    
       return `
-        <div class="transaction-detail">
+        <li class="transaction-detail">
           <div class="transaction-info">
             <div class="datetime">
               <p class="date-text">${new Date(transaction.date).getDate()}</p>
               <div>
                 <div class="daymonthyear">
-                  <p class="daymonthyear-text">${transactionDate[2]},</p>
+                  <p class="daymonthyear-text">${transactionDate[2]}</p>
                   <p class="daymonthyear-text">${ transactionDate[0]}</p>
                   <p class= "daymonthyear-text">${transactionDate[1]}</p>
                 </div>
                 <p class="note-text">${transaction.note}</p>
                </div>
             </div>
             <p class="amount-${amountClass}">${amountSign}$ ${formatNumber(
-              Math.abs(transaction.outflow),
+              Math.abs(transaction.outflow)
             )}</p>
           </div>
-        </div>
+        </li>
         `;
     }
 
     static displayCategory = (category, transactions, totalOutFlow) => { 
       const isIncome = totalOutFlow >= 0;
       const totalOutFlowSign = isIncome ? '+' : '-';
+      const transaction = transactions.map(transaction => TransactionTemplate.displayTransaction(transaction));
```

Comment:
Please choose another name, map return array, not item

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks